CHANGELOG.md

```diff
@@ -14,6 +14,13 @@ Releases
 This high level changelog is usually updated when a release is tagged.
 On the master branch there may be changes that are not (yet) described here.
 
+### 1.21
```

Comment:
Could you use `1.21.0` here, and in the commit title? For consistency with all previous releases, and to easily differentiate with future `1.21.x` versions.

Reply:
Fixed.

Reply:
:pray: